force-app/main/default/lwc/marketingCloudCopadoDataTable/marketingCloudCopadoDataTable.js

```diff
@@ -264,12 +255,17 @@ export default class MarketingCloudCopadoDataTable extends LightningElement {
 
     // This Apex method gets the metadata from the last metadata.json File, that was created by the Retrieve Apex method
     try {
-      console.log("Running Initial getMetadataFromEnvironment, this is the userStoryId: ",this.userStoryId);
+      console.log(
+        "Running Initial getMetadataFromEnvironment, this is the userStoryId: ",
+        this.userStoryId
+      );
       getMetadataFromEnvironment({ userStoryId: this.userStoryId }).then(
         (result) => {
           const parsedResult = JSON.parse(result);
           this.data = parsedResult;
-          this.visibleData = parsedResult;
+          // 68 sort the data then assign table variable
+          this.sortData(this.sortedBy, this.sortDirection);
+          //  self.visibleData = parsedResult;
```

Comment:
```suggestion
          this.sortData(this.sortedBy, this.sortDirection);
```
removing commented code